### PR TITLE
ADD: r-pythoninr

### PR DESCRIPTION
**Pull request by gwaygenomics**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/r-pythoninr/meta.yaml [hunk 1]
```diff
@@ -17,14 +17,14 @@ build:
 
 requirements:
   build:
-    - python ==3.5
+    - python >=3.5
```
Comment (bgruening) [inline recipes/r-pythoninr/meta.yaml]: please skip python build in the build section with `skip: True  # [py34]`

**Comment by gwaygenomics** [comment]
@bgruening @daler are you familiar with this error:

```sh
[Jun  1 20:36:55] ERRO Task processing failed: Unexpected exit code [1] of container [3f61e4781473 step-4acb0d8053], container preserved
involucro -v=2 -f /anaconda/lib/python3.5/site-packages/galaxy_lib-17.5.9-py3.5.egg/galaxy/tools/deps/mulled/invfile.lua -set CHANNELS='bioconda,file://anaconda/conda-bld,r,defaults,conda-forge' -set TEST='Rscript -e "library('PythonInR')"' -set TARGETS='r-pythoninr=0.1_3=r3.3.1_0' -set REPO='quay.io/biocontainers/r-pythoninr:0.1_3--r3.3.1_0' -set BINDS='build/dist:/usr/local/,/anaconda/conda-bld:/anaconda/conda-bld' build-and-test
Traceback (most recent call last):
  File "/anaconda/bin/mulled-build", line 6, in <module>
    sys.exit(galaxy.tools.deps.mulled.mulled_build.main())
  File "/anaconda/lib/python3.5/site-packages/galaxy_lib-17.5.9-py3.5.egg/galaxy/tools/deps/mulled/mulled_build.py", line 319, in main
    sys.exit(mull_targets(targets, **args_to_mull_targets_kwds(args)))
  File "/anaconda/lib/python3.5/site-packages/galaxy_lib-17.5.9-py3.5.egg/galaxy/tools/deps/mulled/mulled_build.py", line 172, in mull_targets
    return involucro_context.exec_command(involucro_args)
  File "/anaconda/lib/python3.5/site-packages/galaxy_lib-17.5.9-py3.5.egg/galaxy/tools/deps/mulled/mulled_build.py", line 204, in exec_command
    os.rmdir('./build')
OSError: [Errno 39] Directory not empty: './build'
```

It looks like it could be an issue with the travis build.

**Comment by bgruening** [comment]
@gwaygenomics yes this is when a previous process was killed and some docker artefacts are left over. Was this locally or on travis.

**Comment by gwaygenomics** [comment]
Travis build logs

**Comment by gwaygenomics** [comment]
@bgruening - yes, the error was found on travis, starting on [line 615](https://travis-ci.org/bioconda/bioconda-recipes/jobs/238494555)

**Comment by bgruening** [comment]
Wired, can you merge master into your branch and try again?

**Comment by johanneskoester** [comment]
We  have made changes to our build system (migrated from Travis CI to  Circle CI). If you would like to continue working on this PR, please  reopen this PR and update with latest upstream/master.

**Comment by gwaygenomics** [comment]
updated build in conda-forge/staged-recipes#5215

**Pull request closed without merging**